README.md

```diff
@@ -1,24 +1,911 @@
+      <li>Mutation endpoint(s) for creation of new encounters</li>
+      <li>Current deploy GraphQL API url: https://tavern-keeper-be.onrender.com/</li>
+    </ul>
+    <br />
+    <a href="https://github.com/Tavern-Keeper-2308/tavern_keeper_be"><strong>Explore Backend docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/Tavern-Keeper-2308/tavern-keeper-fe">Frontend Repository</a>
+    ·
+    <a href="https://github.com/Tavern-Keeper-2308/tavern_keeper_be/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/Tavern-Keeper-2308/tavern_keeper_be/issues">Request Feature</a>
+  </p>
+</div>
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+  1. [About The Project](#about-the-project)
+      - [Built With](#built-with)
+      - [Gems](#gems)
+  2. [Getting Started](#getting-started)
+      - [Installation](#installation)
+      - [Testing](#testing)
+  3. [Database](#database)
+  4. [Endpoints](#endpoints)
+      - [Query: getMonsters](#query-getmonsters)
+      - [Query: getMonster](#query-getmonster)
+      - [Query: getEncounters](#query-getencounters)
+      - [Query: getEncounter](#query-getencounter)
+      - [Mutation: createEncounter](#mutation-createencounter)
+  5. [Roadmap](#roadmap)
+  6. [Contributors](#contributors)
 
-* ...
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+<img src="images/landing.png" alt="landing" width="400" height="200">
+
+[![Tavern Keeper](https://img.shields.io/badge/Tavern_Keeper-Visit_Website-purple?style=for-the-badge)](INSERT-DEPLOY-LINK)
+
+### Built With
+* [![Ruby on Rails][Rails-shield]][Rails-url]
+* [![GraphQL][GraphQL-shield]][GraphQL-url]
+* [![PostgreSQL][PostgreSQL-shield]][PostgreSQL-url]
+* [![HTML][HTML-shield]][HTML-url]
+* [![JavaScript][JavaScript-shield]][JavaScript-url]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Gems
+
+#### Deployment
+* [![faraday][gem-faraday]][gem-faraday-url]
+* [![jsonapi-serializer][gem-jsonapi-serializer]][gem-jsonapi-serializer-url]
+
+#### Testing
+* [![debug][gem-debug]][gem-debug-url]
+* [![rspec-rails][gem-rspec-rails]][gem-rspec-rails-url]
+* [![simplecov][gem-simplecov]][gem-simplecov-url]
+* [![factory_bot_rails][gem-factory_bot_rails]][gem-factory_bot_rails-url]
+* [![faker][gem-faker]][gem-faker-url]
+* [![pry][gem-pry]][gem-pry-url]
+* [![shoulda-matchers][gem-shoulda-matchers]][gem-shoulda-matchers-url]
+* [![capybara][gem-capybara]][gem-capybara-url]
+* [![webmock][gem-webmock]][gem-webmock-url]
+* [![vcr][gem-vcr]][gem-vcr-url]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Database 
+
+<img src="images/schema.png"  width="300" height="800">
```

Comment:
This image is also not loading and doesn't appear to exist in the images folder.